jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -677,6 +684,8 @@ public final ThreadGroup getThreadGroup() {
 
 /**
  * Posts an interrupt request to the receiver
+ * 
+ * From Java 14, the interrupt state for threads that are not alive is tracked.
```

Comment:
Pls use the preprocessor to only include this in Java 14+. Similarly for isInterrupted().